vector/src/main/java/im/vector/app/features/settings/devices/v2/overview/SessionOverviewViewModel.kt

```diff
@@ -213,10 +232,16 @@ class SessionOverviewViewModel @AssistedInject constructor(
 
     private fun handleTogglePusherAction(action: SessionOverviewAction.TogglePushNotifications) {
         viewModelScope.launch {
-            val devicePushers = awaitState().pushers.invoke()?.filter { it.deviceId == action.deviceId }
-            devicePushers?.forEach { pusher ->
+            val devicePusher = session.pushersService().getPushers().firstOrNull { it.deviceId == initialState.deviceId }
+            devicePusher?.let { pusher ->
                 session.pushersService().togglePusher(pusher, action.enabled)
             }
+
+            val newNotificationSettingsContent = mapOf("is_silenced" to !action.enabled)
```

Comment:
Is it possible to use the `LocalNotificationSettingsContent` instead of the map with the `toContent` extension method?